thrift/history.thrift

```diff
@@ -871,10 +871,8 @@ service HistoryService {
     throws (
       1: shared.BadRequestError badRequestError,
       2: shared.InternalServiceError internalServiceError,
-      3: shared.EntityNotExistsError entityNotExistError,
```

Comment:
I think this error still applies. If the API is called on a workflow that doesn't exist, this error should be returned. It's different from the no-op case when the workflow has already closed.